kong/runloop/handler.lua

```diff
@@ -813,11 +813,15 @@ return {
         balancer.init()
       end)
 
-      timer_every(1, function(premature)
+      local router_update_frequency = kong.configuration.router_update_frequency or 1
+
+      timer_every(router_update_frequency, function(premature)
         if premature then
           return
         end
 
+        kong.log.err("updating router #", ngx.worker.id())
```

Comment:
This info does not seems to warrant an `error`.

Reply:
oh, a leftover.